README.md

```diff
@@ -171,17 +152,12 @@ except for the fact that it is not local anymore.
 
 Similarly, if you do not want to use anything other than local storage, you can still use Mayastor to provide you with
 additional functionality that otherwise would require you setup kernel specific features like LVM for example.
-<br>
-
-</p>
 
 ## Exporting the Nexus
 
 <p align="justify">
 The primary focus of development is using NVMe as a transport protocol. The Nexus uses
 NVMe-oF to replicate a volume's data to multiple devices on multiple nodes (if required).
-
-<br>
```

Comment:
Does it not look too close client without the break?

Reply:
Seems fine to me.